File: Play With Me/PWMApi/PWMApi/Models/Game.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace PWMApi.Models
{
    public class Game
    {
        public long Id { get; set; }
        public string Naam { get; set; }
        public long MinSpelers { get; set; }
        public long MaxSpelers { get; set; }
        [NotMapped]
        public string[] Tags { get; set; }
    }
}

File: Play With Me/playapp/src/app/game/game.ts
export class Game {


    constructor(
        private _id: number,
        private _naam: string,
        private _minSpelers: number,
        private _maxSpelers: number,
        private _tags: string[]
    ) {

    }



    get id(): number {
        return this._id;
    }
    get naam(): string {
        return this._naam;
    }
    get minSpelers(): number {
        return this._minSpelers;
    }
    get maxSpelers(): number {
        return this._maxSpelers;
    }
    get tags(): string[] {
        return this._tags;
    }



    static fromJSON(json: any): Game {
        const g = new Game(json.id, json.naam, json.minSpelers, json.maxSpelers, json.tags)
        return g;
    }

}